iron-multi-selectable.html

```diff
@@ -108,6 +115,12 @@
         this.splice('selectedValues',i,1);
       }
       this._selection.setItemSelected(this._valueToItem(value), unselected);
+    },
+
+    _valuesToItems: function(values) {
+      return (values == null) ? null : values.map(function(value) {
```

Comment:
Oh, I'm not saying we should remove this function - mirroring IronSelectableBehavior's internals is totally a good thing here. I'm just curious about preferences between `thisArg` and binding; not important though.